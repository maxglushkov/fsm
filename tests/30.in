3
2
SF b SF
SF b AM
AM a SF
AM a AM
S b SF
S a SF
S a AM
