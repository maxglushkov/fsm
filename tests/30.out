3
2
S0S b S1F
S0S a S2F
S1F b S2F
S2F b S2F
S2F a S2F
